support_branding/static/src/xml/base.xml

```diff
@@ -1,36 +1,64 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates>
     <t t-extend="CrashManager.error">
-        <t t-jquery="div" t-operation="after">
-            <div>
+        <t t-jquery="div.o_error_detail" t-operation="after">
+            <div class="container">
                 <form
                     class="support-branding-submit-form oe_form"
                     action="mailto:"
                     enctype="text/plain"
                     method="get"
                 >
```

Comment:
In my Chrome I'm getting this message:

![image](https://user-images.githubusercontent.com/1466356/126187060-7057fd52-4d40-4a4b-9496-ccbca91756a3.png)

Could this be suppressed somehow? It also eclipses the 'Email to **company**' button

Reply:
for this wasn't able to recreate it on my end but added autocomplete=off as way to suppress also check version of chrome you are using or if any plugins might affect.

Reply:
The autocomplete off didn't work for me, did some research and they suggest to remove the "mailto:" and use a roundabout way to do it by Javascript:

Reply:
So the issue was due to this line:
`$form.attr("action", "mailto:" + self.support_cp_email);`

So removed it so that we rely on send the mail via `mail.mail` on odoo. This just launches the the form later on submission by getting the default mail app installed.

Reply:
Ahh great it now works. Didnt realize that we didnt need `mailto:` at all.